app/pods/publications/publication/template.hbs

```diff
@@ -10,7 +10,14 @@
               this.casePath
             }}
           </span>
-          <WebComponents::AuAbbreviatedText @text={{this.titleText}} @size="150" @custom="true" />
+          <WebComponents::AuAbbreviatedText
+            data-test-publication-detail-menu-short-title
+            @text={{this.titleText}}
+            @size="150"
+            @custom="true"
+            @customTextClass="auk-toolbar-complex__title"
+          >
+          </WebComponents::AuAbbreviatedText>
```

Comment:
Data-tag stond in de AuAbbreviatedText maar was de foute plaats
customTextClass kan je meegeven en we gaan die css gebruiken indien @custom="true"

Reply:
Voor booleans mag je met een echte boolean werken: `@custom={{true}}`